src/components/actions/userFoodTicketActions.ts

```diff
@@ -1,15 +1,15 @@
 "use server";
 
-import type { FoodEventCreateFormInputValues } from "@/../ui-components/FoodEventCreateForm";
+import type { FoodEventCreateFormInputValues } from "@../../../ui-components/FoodEventCreateForm";
 import client from "@/components/_Amplify/AmplifyBackendClient";
 
 export async function createFoodEvent(fields: FoodEventCreateFormInputValues) {
   const { errors } = await client.models.FoodEvent.create({
-    name: fields.name,
-    description: fields.description,
-    start: fields.start,
-    end: fields.end,
-    totalGroupCount: fields.totalGroupCount,
+    name: fields.name ? fields.name : "",
+    description: fields.description ? fields.description : "",
+    start: fields.start ? fields.start : "",
```

Comment:
is there a reason these files are changed?

Reply:
Yeah, I had to change this. I think since we updated Amplify, when I was trying to use the testing form, it didn't work anymore and said "Type 'string | undefined' is not assignable to type 'string'."
If we can leave this as is I think we can delete this testing stuff later when we're done (but it's probably useful for testing since food events are really hard to create normally)

Reply:
I can just delete this file tho cuz its testing stuff